src/app/_shared-components/PostDetails/VoteReferendum/VoteReferendum.tsx

```
@@ -92,8 +252,15 @@ function VoteReferendum({ index }: { index: string }) {
 				small
 				customAddressSelector={<AddressRelationsPicker withBalance />}
 			/>
+			{delegatedVotingPower && delegatedVotingPower.gt(BN_ZERO) && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)